src/irmin/tree.ml

```diff
@@ -991,7 +980,11 @@ module Make (P : Private.S) = struct
   let fold ?(force = `And_clear) ?(uniq = `False) ?(pre = id) ?(post = id)
       ?depth ?(contents = id) ?(node = id) (t : t) acc =
     match t with
-    | `Contents v -> contents Path.empty (fst v) acc
+    | `Contents (c, _) -> (
+        (* TODO: respect {!force} here? *)
```

Comment:
I think we should probably respect the `force` flag here, but haven't done so yet since it's another breaking change. If not, we should have a proper error message.

@samoht: WDYT?

Reply:
AFAIK Force:flag is not used in production so feel free to fix the associated behavior

Reply:
Done :+1: